shell/spy.go

```diff
@@ -1,19 +1,32 @@
 package shell
 
 import (
-	"bytes"
+	"fmt"
+	"io"
 	"log"
 	"os/exec"
+	"strings"
 )
 
-// Execute accepts a command argument and returns the result of the executed command
-func Execute(script string) string {
-	cmd := exec.Command(script)
-	var out bytes.Buffer
-	cmd.Stdout = &out
+// ParseCommand accepts a string and returns the command name and args
+func ParseCommand(cmd string) (string, []string) {
+	stringResult := strings.Split(cmd, " ")
```

Comment:
This doesn't do quite what we want in the case of strings that contain quotes; I recommend `shell.Split` from [`bitbucket.org/creachadair/shell`](https://bitbucket.org/creachadair/shell/src/master/) here (see [this commit](https://github.com/bitfield/script/commit/cc3ab5e81e4cbec0882631bb20373403955e5595) for an example of where and why it's useful).